x/claims/keeper/abci_test.go

```diff
@@ -154,6 +172,10 @@ func (suite *KeeperTestSuite) TestClawbackEmptyAccounts() {
 			moduleAcc := suite.app.AccountKeeper.GetModuleAccount(suite.ctx, distrtypes.ModuleName)
 			balance := suite.app.BankKeeper.GetBalance(suite.ctx, moduleAcc.GetAddress(), "aevmos")
 			suite.Require().Equal(tc.expBalance, balance.Amount.Int64())
+
+			// test that all claims records are deleted
+			claimsRecords := suite.app.ClaimsKeeper.GetClaimsRecords(suite.ctx)
+			suite.Require().Len(claimsRecords, 0)
```

Comment:
If this keeper is mutating the accounts state that should also be tested, no? Apologies if I am missing something here